src/sql/src/plan/query.rs

```diff
@@ -2752,21 +2752,22 @@ fn plan_aggregate(
             (args, order_by.clone())
         }
     };
-    let (order_by_exprs, map_exprs) = plan_order_by_exprs(ecx, &order_by)?;
 
-    // order_by contains column indexes for the containing scope, and we need to
```

Comment:
I think some variation of this comment is worth leaving. Maybe @mjibson can wordsmith a version appropriate for this new version?

Reply:
I think the comment is still accurate and could be moved/restored? I don't care too much either way.